src/providers/SpeechProvider/SpeechProvider.actions.js

```diff
@@ -177,19 +178,35 @@ export function getVoices() {
       const regex = new RegExp('^[a-zA-Z]{2,}-$', 'g');
       const fvoices = pvoices.filter(voice => !regex.test(voice.lang));
       voices = fvoices.map(
-        ({ voiceURI, lang, name, Locale, ShortName, DisplayName, Gender }) => {
+        ({
+          voiceURI,
+          lang,
+          name,
+          Locale,
+          ShortName,
+          DisplayName,
+          Gender,
+          localService,
+          features
+        }) => {
           let voice = {};
           if (lang) {
             voice.lang = lang;
           } else if (Locale) {
             voice.lang = Locale;
           }
-          if (voiceURI) {
+          if (isAndroid() && features && /notInstalled/.test(features)) {
+            voice.voiceURI = voiceURI;
+            voice.voiceSource = 'cloud';
+            voice.isAzure = false;
+          } else if (voiceURI && localService) {
```

Comment:
@RodriSanchez1 The meaning of the `localService` properly is not that a voice is from a cloud service or not. The meaning of the property indicates if the voice comes from a remote service or not. In this context, a voice provided by the browser is remote, while a voice provided by the operating system will be a local voice. 
https://wicg.github.io/speech-api/#dom-speechsynthesisvoice-localservice

Reply:
https://developer.chrome.com/blog/web-apps-that-talk-introduction-to-the-speech-synthesis-api/